cmake/OpenCVUtils.cmake

```diff
@@ -1396,37 +1396,16 @@ macro(ocv_parse_header FILENAME FILE_VAR)
 endmacro()
 
 # read single version define from the header file
+# set ${LIBNAME}_VERSION_STRING to VARNAME without quotes
 macro(ocv_parse_header2 LIBNAME HDR_PATH VARNAME)
-  ocv_clear_vars(${LIBNAME}_VERSION_MAJOR
-                 ${LIBNAME}_VERSION_MAJOR
-                 ${LIBNAME}_VERSION_MINOR
-                 ${LIBNAME}_VERSION_PATCH
-                 ${LIBNAME}_VERSION_TWEAK
-                 ${LIBNAME}_VERSION_STRING)
+  ocv_clear_vars(${LIBNAME}_VERSION_STRING)
```

Comment:
Keep compatibility!
This macro is used not for ZLIB only.

Create a new one instead, e.g. `ocv_parse_header_version` and use it.

Reply:
Hi, thanks for the review. I add a new macro `ocv_parse_header_version` . However, I don't think there is compatiblity problem. There are only two libraries zlib and japser using `ocv_parse_header2`. Zlib and japser only use ${LIBNAME}_VERSION_STRING not ${LIBNAME}_VERSION_MAJOR etc. I think it's safe to delete `ocv_parse_header2`.